.github/workflows/ubuntu.yml

```diff
@@ -1,10 +1,10 @@
-name: Python application
+name: Ubuntu
 
 on: [push, pull_request]
```

Comment:
What is the best practice here? Does it make sense to trigger the build on both "push" and "pull_request"?